blueprints/component-addon/index.js

```diff
@@ -11,13 +11,13 @@ module.exports = {
   fileMapTokens: function() {
     return {
       __path__: function(options) {
-        if (options.pod) {
+        if (options.pod && options.inRepoAddon) {
```

Comment:
The generator handles the path from the app component to the addon component, so we don't need to include the pods path in the app.  This has a side benefit of not requiring the consuming application to use a matching podModulePrefix.